idunn/api/closest.py

```diff
@@ -23,16 +24,20 @@ def get_closest_place(lat: float, lon: float, es: Elasticsearch):
 
     if loader is None:
         prometheus.exception("FoundPlaceWithWrongType")
-        raise Exception("Closest address to '{}:{}' has a wrong type: '{}'".format(lat, lon, es_addr.get('_type')))
+        raise HTTPException(
+            status_code=404,
+            detail="Closest address to '{}:{}' has a wrong type: '{}'".format(lat, lon, es_addr.get('_type')))
```

Comment:
As this error would show a problem in the database, it should not fail silently.  
We should keep a warning message at least.

Reply:
Isn't it why we have `prometheus.exception("FoundPlaceWithWrongType")`?

Reply:
Prometheus would not give any detail about the query that caused the error. It's only used to build metrics.

In this case where the error is quite unexpected and may bring another issue to our attention, an error message is much more useful than a counter.